src/bundle/Resources/config/default_settings.yaml

```diff
@@ -56,3 +56,12 @@ parameters:
         template: '@@IbexaFieldTypeRichText/RichText/embed/location_inline.html.twig'
     ibexa.site_access.config.default.fieldtypes.ezrichtext.embed.location_inline_denied:
         template: '@@IbexaFieldTypeRichText/RichText/embed/location_inline_denied.html.twig'
+
+    # RichText namespaces map to migrate
+    ibexa.field_type.rich_text.namespaces_migration_map:
+        'xmlns:ezxhtml="http://ez.no/xmlns/ezpublish/docbook/xhtml"': 'xmlns:ezxhtml="http://ibexa.co/xmlns/dxp/docbook/xhtml"'
```

Comment:
Yes, it is still in use

Reply:
I am aware, but the question is if we shouldn't aim to rebrand it as well? ;) It might be out of scope of this PR, but the question still stands, POV ping @alongosz.

Reply:
> I am aware, but the question is if we shouldn't aim to rebrand it as well? ;) It might be out of scope of this PR, but the question still stands, POV ping @alongosz.

Yes if technically possible. I'm not sure how this would affect OE @ibexa/javascript-dev

Reply:
I think it doesn't matter for the CKEditor.

Reply:
>> Should `ezxhtml` be still in use?

> Yes if technically possible. I'm not sure how this would affect OE

Okay, so it indeed is a backend issue not an OE one. The same way we updated `xmlns` URI we should update XML namespace names:

- `xmlns:ez`
- `xmlns:ezxhtml`
- `xmlns:ezcustom`

to be eg. `ibx` and provide upgrade script like this one (ideally just extending it). It would be good to include it right now to reduce possible future complexity, but it's not as simple as just upgrading field values in the database. The entire `ezpublish.rng` needs to be refactored, including also other legacy naming occurrences (not to mention file name itself, which is a BC issue).

FYI @webhdx @adamwojs maybe we need to schedule additional follow-up work for this.